content/manuals/harmonia/_index.md

```diff
@@ -263,7 +263,9 @@ Run `docker harmonia doctor` to print helpful troubleshooting information.
 - KinD does not run on Project Harmonia due to some hard-coded assumptions to ensure it's running in a privileged container. K3d is a good alternative.
 - Containers cannot access host through DNS `host.docker.internal`.
 - File binds (non-directory binds) are currently static, meaning changes will not be reflected until the container is restarted. This also affects Compose configs and secrets directives.
-- Bind mounts, such as `-v /localpath:/incontainer` in the `docker run` command, are not supported without using a file-sync.
+- Bind _mounts_, such as `-v /localpath:/incontainer` in the `docker run` command, require starting a file-sync.
+- Starting a file-sync for a directory with a large amount of may take extra time to sync and become ready for us in a container.
```

Comment:
```suggestion
- Starting a file-sync for a directory with a large amount of may take extra time to sync and become ready for use in a container.
```

Reply:
"a large amount *of data/files" may take..."

Reply:
@stevvooe, the above suggestion didn't go in...

- Creating a synchronized file share for a directory with a large amount of [**missing noun**] may take extra time to sync and become ready for use in a container.